engines/optix/OptiXEngine.cpp

```diff
@@ -181,6 +182,25 @@ void OptiXEngine::_createRenderers()
         context.addRenderer("basic", osp);
         addRendererType("basic");
     }
+
+    { // Basic simulation
```

Comment:
Do you mind reviewing the AdvancedSimulation renderer? It seems that it's not really implemented and all the parameters in this file are useless. If you can confirm it's the case, remove it.